src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -24,6 +23,15 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
 {
     public abstract partial class Frame : IFrameControl
     {
+        // byte consts don't have a data type annotation so we pre-cast them
```

Comment:
Wouldn't you have to explicitly cast from char to byte for non-const field declarations?